libs/dh/wholesale/feature-settlement-reports/src/lib/table/dh-settlement-reports-table.component.ts

```diff
@@ -117,25 +119,21 @@ export class DhSettlementReportsTableComponent {
       return;
     }
 
-    const fileName = this.settlementReportName(settlementReport);
-    const fileOptions = { name: fileName, type: 'application/zip' };
+    settlementReportDownloadUrl = `${settlementReportDownloadUrl}&filename=${this.settlementReportName(settlementReport)}&fromApi=${settlementReport.fromApi}`;
```

Comment:
suggestion: trying the download in T-001 env, it seems like the `fromApi` param already exists. No need to add it again 🤔

![image](https://github.com/user-attachments/assets/809628c3-f612-409d-a163-b58e93178019)

Reply:
```suggestion
    settlementReportDownloadUrl = `${settlementReportDownloadUrl}&filename=${this.settlementReportName(settlementReport)}`;
```